qemu/tests/cfg/balloon_in_use.cfg

```diff
@@ -14,22 +14,11 @@
     balloon_timeout = 240
     check_setup_events = balloon_test_setup_ready
     Windows:
-        x86_64:
-            program_files = "%ProgramFiles(x86)%"
-        i386:
-            program_files = "%ProgramFiles%"
         stress_test = win_video_play
-        #Disable first startup guide for windows media player
-        wmplayer_reg_cmd = "reg add HKLM\SOFTWARE\Policies\Microsoft\WindowsMediaPlayer /v GroupPrivacyAcceptance  /t REG_DWORD /f /d 00000001"
-        wmplayer_path = "${program_files}\Windows Media Player\wmplayer.exe"
-        #Install kmplayer if wmplayer is not installed default
-        kmplayer_install_cmd = "start /wait WIN_UTILS:\kmplayer\%s\KMPlayer-setup.exe /SP- /VERYSILENT"
-        kmplayer_path = "${program_files}\KMPlayer\kmplayer.exe"
+        mplayer_path = "WIN_UTILS:\mplayer.exe"
+        play_video_cmd = "start /MIN %s %s -loop 0 -fs"
```

Comment:
Need a param "target_process" for cases "during_bg_test". Otherwise,it will always get True status when check_bg_running in driver_in_use.py.
```
 24     def check_bg_running(target_process):                                       
 32         session = vm.wait_for_login()                                           
 33         list_cmd = params.get("list_cmd", "wmic process get name")              
 34         output = session.cmd_output_safe(list_cmd, timeout=60)                  
 35         process = re.findall(target_process, output, re.M | re.I)               
 36         session.close()                                                         
 37         return bool(process)  
```

Reply:
@yanan-fu Yeah, you are right, thanks!